src/constants/index.ts

```diff
@@ -1,7 +1,3 @@
-export const PARTICIPATE = 'participate'
-export const PARTICIPATE_ERROR = 'participate-error'
-export const RESET_PARTICIPANTS = 'reset-participants'
-
 export const APPLY = 'apply'
```

Comment:
why? i c you moving them into the action creator files. seems like a recipe for eventually needing to move them back

Reply:
We can leave them in constants if you want, but in codebases I have worked on in the past, I have never had a need for the action type constants to be exposed outside of the action creators and the reducers (besides in specs, of course). It appears this is so far true in this codebase as well.

By moving the action types into the action creator files, I am trying to reduce the amount of constants that are actually used globally, making it a bit easier to read through the file.